packages/core/src/Stores/notification-store.js

```diff
@@ -1130,6 +1135,16 @@ export default class NotificationStore extends BaseStore {
                 type: 'announce',
                 should_hide_close_btn: false,
             },
+            poa_older_than_six_months: {
```

Comment:
maybe you can add some test cases for this. 🧪

Reply:
Added in the card their are no test files for stores